bazel-haskell-deps.bzl

```diff
@@ -416,6 +417,7 @@ exports_files(["stack.exe"], visibility = ["//visibility:public"])
             "ghc-boot-th",
             "ghc-lib",
             "ghc-lib-parser",
+            "ghc-lib-parser-ex",
```

Comment:
What is this new library?

Reply:
Bits of code that interact with ghc-lib-parser have been factored out into a new hlint dependency, ghc-lib-parser-ex.